app/test/unit/create-branch-test.ts

```diff
@@ -20,6 +20,7 @@ const stubTip = {
   authoredByCommitter: true,
   parentSHAs: [],
   trailers: [],
+  tags: null,
```

Comment:
Should this be an empty array?

```suggestion
  tags: [],
```

Reply:
ah good catch! this type is totally wrong since this is now an `IBranchTip` instead of a `Commit`. fixing!

Reply:
fixed in 4be73cf